.prettierrc

```diff
@@ -1,7 +1,3 @@
 {
-  "endOfLine": "lf",
-  "semi": true,
-  "singleQuote": true,
-  "tabWidth": 2,
-  "trailingComma": "es5"
+  "singleQuote": true
```

Comment:
Why did we remove these rules?, I have added them to the website

Reply:
These are the default values as mentioned on: https://github.com/dislearning/website/pull/1#discussion_r512072501